src/app/parcels/parcelsTable/ParcelsTable.tsx

```diff
@@ -225,22 +224,17 @@ const ParcelsTable: React.FC<ParcelsTableProps> = ({
         }
     }, [areFiltersLoadingForFirstTime, fetchAndDisplayParcelsData]);
 
-    const packingManagerViewDataPortion = React.useCallback(
-        (parcelsDataPortion: ParcelsTableRow[]): ParcelsTableRow[] => {
-            return parcelsDataPortion.filter((parcel) => {
-                if (shouldBeInPackingManagerView(parcel, today, yesterday)) {
-                    return parcel;
-                }
-            });
-        },
-        [today, yesterday]
-    );
+    const packingManagerViewDataPortion = useMemo(() => {
+        return parcelsDataPortion.filter((parcel) => {
+            if (shouldBeInPackingManagerView(parcel, today, yesterday)) {
+                return parcel;
+            }
+        });
+    }, [parcelsDataPortion, today, yesterday]);
```

Comment:
`Array.filter` works on boolean return types from the filter function, and if there's only one statement in a function we can remove the curly brackets and return statement. Overall, this means we can simplify this a bit more:
```
const packingManagerViewDataPortion = useMemo(() =>
    parcelsDataPortion.filter((parcel) => shouldBeInPackingManagerView(parcel, today, yesterday))
    );
}, [parcelsDataPortion, today, yesterday]);
```